apache-hbase-mixin/dashboards/apache-hbase-logs-overview.libsonnet

```diff
@@ -0,0 +1,32 @@
+local g = import 'github.com/grafana/grafonnet/gen/grafonnet-latest/main.libsonnet';
+local logsDashboard = import 'github.com/grafana/jsonnet-libs/logs-lib/logs/main.libsonnet';
+{
+  grafanaDashboards+::
+    if $._config.enableLokiLogs then {
+      local apacheHBaseLogs =
+        logsDashboard.new(
+          'Apache HBase logs overview',
+          datasourceName='loki_datasource',
+          datasourceRegex='(?!grafanacloud.+usage-insights|grafanacloud.+alert-state-history).+',
```

Comment:
This is not required here. Such filters are applied automatically on integrations phase, by one of the utils functions.